engine/src/main/java/org/terasology/engine/core/module/ModuleManager.java

```diff
@@ -265,6 +290,30 @@ public ModuleFactory getModuleFactory() {
     }
 
     public PermissionProvider getPermissionProvider(Module module) {
-        return permissionProviderFactory.createPermissionProviderFor(module, module.getClassPredicate());
+        // TODO: This has only one usage. Audit to see if it's worth keeping.
+        return permissionProviderFactory.createPermissionProviderFor(
+                module,
+                x -> false  // modules (other than engine) never contain classpath classes
```

Comment:
This is potentially a functional change, but probably not. The method doesn't seem to be much used, and `false` is probably the thing game modules have in this field anyway.